docs/Makefile

```diff
@@ -1,40 +1,49 @@
-LOCAL_ROOT = $(git rev-parse --show-toplevel)
-SHELL = /bin/bash
-THEME_MODULE = gitlab.com/f5/nginx/controller/poc/f5-hugo
-# export THEME_PATH="</local/path/to/theme>"
-THEME_PATH = $(LOCAL_THEME_PATH)
-THEME_BRANCH = development
+THEME_MODULE = github.com/nginxinc/nginx-hugo-theme
+## Pulls the current theme version from the Netlify settings
+THEME_VERSION = $(NGINX_THEME_VERSION)
 
-.PHONY: all all-local clean hugo-mod build-production build-staging hugo-server-drafts hugo-server
+.PHONY: all clean hugo-mod build-production build-staging hugo-server-drafts hugo-server netlify
 
 all: hugo-mod build-production
 
-all-local: clean hugo-mod build-production
+all-staging: hugo-mod build-staging
 
+all-dev: hugo-mod build-dev
+
+# Removes the public directory generated by the `hugo` command
 clean:
-	rm -rf ${LOCAL_ROOT}/public
+	if [[ -d ${PWD}/public ]] ; then rm -rf ${PWD}/public && echo "Removed public directory" ; else echo "Did not find a public directory to remove" ; fi
 
 hugo-mod:
-	hugo mod clean
-	rm -rf _vendor
-	hugo mod get
-	hugo mod vendor
+	hugo mod get $(THEME_MODULE)@v$(THEME_VERSION)
 
+# Builds using the Hugo "staging" environment
```

Comment:
Typo - should be "production" environment

Reply:
good catch! fixed